test-utils.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import test from 'ava';
+
+const config = require('./config');
+const utils = require('./utils');
+
+const CDN_URL = 'https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/';
```

Comment:
Yeah CDN_URL should be visible somewhere, I agree. I'm going to stick it on config, hopefully that works for you.

As for `ALT_URL` - I shouldn't have called it that in the tests or used a real URL, because it is confusing. It is actually a different `ALT_URL`... Confusing I know -  I think the updated test code will make more sense.

Reply:
Thank you!